packages/react-components/react-charts-preview/library/src/components/CommonComponents/Popover.tsx

```diff
@@ -0,0 +1,247 @@
+    const toDrawShape = xValue.index !== undefined && xValue.index !== -1;
+    const { culture } = props;
+    const yValue = convertToLocaleString(xValue.y, culture);
+    if (!xValue.yAxisCalloutData || typeof xValue.yAxisCalloutData === 'string') {
+      return (
+        <div style={yValueHoverSubCountsExists ? marginStyle : {}}>
+          {yValueHoverSubCountsExists && (
+            <div className="ms-fontWeight-semibold" style={{ fontSize: '12pt' }}>
+              {xValue.legend!} ({yValue})
+            </div>
+          )}
+          <div
+            id={`${index}_${xValue.y}`}
+            className={classes.calloutBlockContainer}
+            style={{
+              marginTop: props.XValue ? '13px' : 'unset',
+              ...(!toDrawShape
+                ? {
+                    borderLeft: `4px solid ${xValue.color}`,
+                  }
+                : {}),
+            }}
+          >
+            {toDrawShape && (
+              <Shape
+                svgProps={{
+                  className: classes.shapeStyles,
+                }}
+                pathProps={{ fill: xValue.color }}
+                shape={Points[xValue.index! % Object.keys(pointTypes).length] as LegendShape}
+              />
+            )}
+            <div
+              className={mergeClasses(
+                classes.calloutBlockContainer,
+                toDrawShape
+                  ? classes.calloutBlockContainertoDrawShapetrue
+                  : classes.calloutBlockContainertoDrawShapefalse,
+              )}
+            >
+              <div className={classes.calloutlegendText}> {xValue.legend}</div>
+              <div className={classes.calloutContentY}>
+                {convertToLocaleString(
+                  xValue.yAxisCalloutData ? xValue.yAxisCalloutData : xValue.y ?? xValue.data,
+                  culture,
+                )}
+              </div>
+            </div>
+          </div>
+        </div>
+      );
+    } else {
+      const subcounts: { [id: string]: number } = xValue.yAxisCalloutData as { [id: string]: number };
+      return (
+        <div style={marginStyle}>
+          <div className="ms-fontWeight-semibold" style={{ fontSize: '12pt' }}>
+            {xValue.legend!} ({yValue})
+          </div>
+          {Object.keys(subcounts).map((subcountName: string) => {
+            return (
+              <div key={subcountName} className={classes.calloutBlockContainer}>
+                <div className={classes.calloutlegendText}> {convertToLocaleString(subcountName, culture)}</div>
+                <div
+                  className={classes.calloutContentY}
+                  style={{ color: props.color ? props.color : tokens.colorNeutralForeground1 }}
+                >
+                  {convertToLocaleString(subcounts[subcountName], culture)}
+                </div>
+              </div>
+            );
+          })}
+        </div>
+      );
+    }
+  }
+});
+
+export default PopoverComponent;
```

Comment:
Fluent v9 doesn't export default on modules and instead exports from the component point.  Not sure if your callers need to be importing default.

Reply:
This was not updated in the Popover component. Removed it.